apertium-tur-uzb.tur-uzb.dix

```diff
@@ -1,4716 +1,4281 @@
+<e>       <p><l>can<s n="n"/></l>                      <r>hayot<s n="n"/></r></p></e>
+<e>       <p><l>can<s n="n"/></l>                      <r>jon<s n="n"/></r></p></e>
+<e>       <p><l>canavar<s n="n"/></l>                  <r>maxluq<s n="n"/></r></p></e>
+<e>       <p><l>casus<s n="n"/></l>                    <r>shpion<s n="n"/></r></p></e>
+<e>       <p><l>cehennem<s n="n"/></l>                 <r>jahannam<s n="n"/></r></p></e>
+<e>       <p><l>cemiyet<s n="n"/></l>                  <r>hamjamiyat<s n="n"/></r></p></e>
+<e>       <p><l>cemiyet<s n="n"/></l>                  <r>jamiyat<s n="n"/></r></p></e>
+<e>       <p><l>cenaze<b/>töreni<s n="n"/></l>         <r>dafn<s n="n"/></r></p></e>
+<e>       <p><l>cep<s n="n"/></l>                      <r>choʻntak<s n="n"/></r></p></e>
+<e>       <p><l>cesaret<s n="n"/></l>                  <r>jasorat<s n="n"/></r></p></e>
+<e>       <p><l>cetvel<s n="n"/></l>                   <r>jadval<s n="n"/></r></p></e>
+<e>       <p><l>cevap<s n="n"/></l>                    <r>javob<s n="n"/></r></p></e>
+<e>       <p><l>cevher<s n="n"/></l>                   <r>ruda<s n="n"/></r></p></e>
+<e>       <p><l>ceza<s n="n"/></l>                     <r>jazo<s n="n"/></r></p></e>
+<e>       <p><l>ceza<s n="n"/></l>                     <r>jiza<s n="n"/></r></p></e>
+<e>       <p><l>cihaz<s n="n"/></l>                    <r>jihoz<s n="n"/></r></p></e>
+<e>       <p><l>cilt<s n="n"/></l>                     <r>jild<s n="n"/></r></p></e>
+<e>       <p><l>cin<s n="n"/></l>                      <r>jin<s n="n"/></r></p></e>
+<e>       <p><l>cinayet<s n="n"/></l>                  <r>qatl<s n="n"/></r></p></e>
+<e>       <p><l>cins<s n="n"/></l>                     <r>jins<s n="n"/></r></p></e>
+<e>       <p><l>cins<s n="n"/></l>                     <r>xil<s n="n"/></r></p></e>
+<e>       <p><l>cinsiyet<s n="n"/></l>                 <r>cinsiyet<s n="n"/></r></p></e>
+<e>       <p><l>cisim<s n="n"/></l>                    <r>jism<s n="n"/></r></p></e>
+<e>       <p><l>civa<s n="n"/></l>                     <r>simob<s n="n"/></r></p></e>
+<e>       <p><l>cıva<s n="n"/></l>                     <r>simob<s n="n"/></r></p></e>
+<e>       <p><l>ciğer<s n="n"/></l>                    <r>jigar<s n="n"/></r></p></e>
+<e>       <p><l>coğrafya<s n="n"/></l>                 <r>geografiya<s n="n"/></r></p></e>
+<e>       <p><l>cumhurbaşkan<s n="n"/></l>             <r>prezident<s n="n"/></r></p></e>
+<e>       <p><l>cumhuriyet<s n="n"/></l>               <r>respublika<s n="n"/></r></p></e>
+<e>       <p><l>cümle<s n="n"/></l>                    <r>jumla<s n="n"/></r></p></e>
+<e>       <p><l>cüret<s n="n"/></l>                    <r>jur’at<s n="n"/></r></p></e>
+<e>       <p><l>dahi<s n="n"/></l>                     <r>dohiy<s n="n"/></r></p></e>
+<e>       <p><l>dahil<s n="n"/></l>                    <r>doxil<s n="n"/></r></p></e>
+<e>       <p><l>daire<s n="n"/></l>                    <r>departa
```

Comment:
"celb" I don't know this word, where did you see it?

Reply:
It was there, I could not find an exact translation for it, but thought maybe there is a word like that as a part of "celb etmek"(or celbetmek?). 
Since it doesn't ring a bell to you, I am commenting it out.